runtime/js/12_io.js

```diff
@@ -109,10 +109,10 @@
 
   const READ_PER_ITER = 32 * 1024;
 
-  async function readAll(r) {
+  async function readAll(r, options) {
```

Comment:
Oh, didn't catch this the first time: `readAll` is a public API (a deprecated one, but still public) at `Deno.readAll`. d.ts should be changed, or the public option should not get the same semantics.

Reply:
I'm fine with either - which of those would you prefer?

Reply:
Let's not expose it publically - makes it easier to change later.

Reply:
Ok, does that mean just not document it (no changes) or should it be guarded (with a symbol or some-such)?

Reply:
Maybe a `readFileInner` and a `readFIle` would work?

Reply:
Done